src/cdstarcat/catalog.py

```diff
@@ -110,7 +112,15 @@ def __init__(self, path, cdstar_url=None, cdstar_user=None, cdstar_pwd=None):
         self.path = pathlib.Path(path)
         self.objects = {}
         if self.path.exists():
-            self.objects = {i: Object.fromdict(i, d) for i, d in load(self.path).items()}
+            if self.path.suffix.lower() == '.zip':
+                with zipfile.ZipFile(self.path, 'r') as z:
```

Comment:
I don't think `ZipFile` accepts `Path` instances - at least not on py3.5. So `str(self.path)` might be safer.